ironfish/src/rpc/routes/chain/getTransactionStream.ts

```diff
@@ -12,30 +13,66 @@ import { ValidationError } from '../../adapters/errors'
 import { ApiNamespace, router } from '../router'
 
 interface Note {
-  amount: string
```

Comment:
This rename is a breaking change to our RPC, but I figured it isn't used by anyone yet (presumably), so I changed it to match assets.